# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/base/PresShell.cpp
# Commit: 0e259884f052
# Full Hash: 0e259884f052230b24601efc3cde3a5ef5d4e6ad
# Author: Matt Woodrow <mwoodrow@mozilla.com>
# Date: 2019-05-01 15:54:31
# Regressor Bug: 1535945
# File Overlap Count: 1
# Description:
#   Bug 1535945 - Don't skip invalidating frames when creating displayports for async scrollable ancestors. r=tnikkel
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D23816
# ==============================================================================

diff -r 732976d3f555 -r 0e259884f052 layout/base/PresShell.cpp
--- a/layout/base/PresShell.cpp	Wed May 01 08:16:44 2019 +0000
+++ b/layout/base/PresShell.cpp	Wed May 01 04:23:07 2019 +0000
@@ -3329,10 +3329,8 @@
  * moving the visual viewport within the layout viewport.
  */
 static void ScrollToShowRect(nsIScrollableFrame* aFrameAsScrollable,
-                             const nsRect& aRect,
-                             ScrollAxis aVertical,
-                             ScrollAxis aHorizontal,
-                             ScrollFlags aScrollFlags) {
+                             const nsRect& aRect, ScrollAxis aVertical,
+                             ScrollAxis aHorizontal, ScrollFlags aScrollFlags) {
   nsPoint scrollPt = aFrameAsScrollable->GetVisualViewportOffset();
   nsRect visibleRect(scrollPt, aFrameAsScrollable->GetVisualViewportSize());
 